streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -137,7 +137,7 @@ public void close(final boolean clean) {
      */
     public List<ConsumerRecord<byte[], byte[]>> update(final TopicPartition partition,
                                                        final List<ConsumerRecord<byte[], byte[]>> records) {
-        log.debug("{} Updating standby replicas of its state store for partition [{}]", logPrefix, partition);
+        log.trace("{} Updating standby replicas of its state store for partition [{}]", logPrefix, partition);
```

Comment:
I only added it for critical paths. If we want to do that everywhere across the board (i.e. beyond Streams) I think it would require a larger and separate pr.